.github/workflows/build-test-verilator.yml

```diff
@@ -113,14 +135,87 @@ jobs:
           path: /opt/riscv
           key: riscv-${{ env.RISCV_VERSION }}-${{ env.SCCACHE_C_CUSTOM_CACHE_BUSTER }}
 
+      - name: Setup riscv path
+        run: |
+          echo /opt/riscv/bin >> $GITHUB_PATH
+
+  build_matrix:
```

Comment:
The actions runner is saying that build_matrix is defined twice.  I suspect there's a merge issue here, do you mind diffing this against dev-goog?

Reply:
Yup, it was a merge issue. Now fixed.